package.json

```diff
@@ -120,11 +120,12 @@
     "lint-staged": "10.1.1",
     "miragejs": "^0.1.40",
     "ncp": "^2.0.0",
-    "node-sass": "^7.0.1",
+    "sass": "^1.50.0",
```

Comment:
sass is a drop in replacement for the deprecated `node-sass`

Reply:
another thing to look at replacing for the rest of our apps